neo/public/stylesheets/main.css

```diff
@@ -31,11 +34,10 @@ body {
 }
 
 time-series {
-  position: absolute;
-  top: 82%;
-  left: 15%;
-  width: 80%;
-  height: 15%;
+  z-index: 1000;
+  pointer-events: none;
```

Comment:
more explanation on the pointer-events: none: In addition to indicating that the element is not the target of mouse events, the value none instructs the mouse event to go "through" the element and target whatever is "underneath" that element instead.